includes/datastream.inc

```diff
@@ -219,6 +225,8 @@ function islandora_rest_get_datastream_properties(AbstractDatastream $datastream
     'controlGroup' => $datastream->controlGroup,
     'created' => (string) $datastream->createdDate,
     'versionable' => (bool) $datastream->versionable,
+    'checksum' => $datastream->checksum,
```

Comment:
Looks like if we add these to the response outright, it's going to trip up the [REST response validation](https://github.com/discoverygarden/islandora_rest/blob/aacb93eaf0e0a586b84bc95dfffebcb64d2af2a6/tests/islandora_rest.test#L228-L230) in the tests, since applying `assertEqual()` to two arrays checks that both arrays precisely match (inclusions and omissions alike). Looks like Travis flagged 9 places this is occurring? Might require investigation.